app/datatables/repository_datatable.rb

```diff
@@ -158,6 +158,7 @@ def get_raw_records
                                  )
                                  .joins(:created_by)
                                  .where(repository: @repository)
+      return @assigned_rows if params[:assigned] == 'assigned'
```

Comment:
Oh right, it's not needed anymore, since we also show assigned column in repositories...